flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalogBase.java

```diff
@@ -318,13 +320,44 @@ public void alterTable(ObjectPath tablePath, CatalogBaseTable newCatalogTable, b
 					throw new TableNotExistException(catalogName, tablePath);
 				}
 			} else {
```

Comment:
Code will be easier to read if we add return in the if branch and remove else branch.

Reply:
agree. Shall we do it separately since it's orthogonal to what this PR is for? I will refactor that in a [hotfix] PR

Reply:
It's probably better to fix it now while it's still warm.

Reply:
Ok. I fixed in source repo and rebased this PR to master, and the rebase took quite some time even though it's a small change.

In the future I think we should follow [Apache Flink Code Style and Quality Guide](https://docs.google.com/document/d/1JGRVK6udBFa8xbyv7ipPQiLEhxebA64f8KKcaAH0UPY/edit#heading=h.i11eha8b46sj). In section ""Separate Refactoring, Cleanup and Independent Changes" it clarifies that such change should be on its own.